packages/ui-components/src/util/toShortAddress.ts

```diff
@@ -0,0 +1,13 @@
+// Copyright 2017-2018 @polkadot/ui-app authors & contributors
+// This software may be modified and distributed under the terms
+// of the Apache-2.0 license. See the LICENSE file for details.
+
+import { AccountId, AccountIndex, Address } from '@polkadot/types';
+
+export default function toShortAddress (_address?: AccountId | AccountIndex | Address | string | null): string {
```

Comment:
Can you add a FIXME here?

I think all UI code that are copied from polkadot/apps could be factorized and should go in https://github.com/polkadot-js/ui